containers/EmpSponsors.ts

```diff
@@ -23,6 +23,10 @@ interface SponsorMap {
   [sponsor: string]: SponsorPositionState;
 }
 
+interface LiquidationMap {
+  [sponsorPlusId: string]: SponsorPositionState;
```

Comment:
Minor naming nit: I'm getting a little confused where we're using the term `id` to refer to `1` vs `0x1e17a75616cd74f5846b1b71622aa8e10ea26cc0-1`. Here, the convention seems to be:
```
// Id = sponsor liquidation id
Id: 1

// Sponsor + id = 0x1e17a75616cd74f5846b1b71622aa8e10ea26cc0-1
sponsorPlusId: 0x1e17a75616cd74f5846b1b71622aa8e10ea26cc0-1
```

However, in the code below, it seems that you're using the following convention:
```
// Id = sponsor address + sponsor liquidation id
Id: 0x1e17a75616cd74f5846b1b71622aa8e10ea26cc0-1
```

Might make sense to just come up with a consistent term for each type of id for clarity.

Reply:
Yeah the issue here is that the subgraph uses `id` to reference `sponsorAddress+ID`. Is this the line that's specifically confusing you, where I key a liquidation object by its returned `liquidation.id`?  If so, then if I set `liquidation.id` to a more clear variable like `const sponsorAddressPlusId` does it make it clearer?

```
                const sponsorAddressPlusId = liquidation.id
                newLiquidations[sponsorAddressPlusId] = {
                  sponsor: utils.getAddress(liquidation.sponsor.id),
                  liquidator: liquidation.liquidator?.address,
                  disputer: liquidation.disputer?.address,
                  liquidationId: liquidation.liquidationId,
                  liquidatedCR: liquidatedCR.toString(),
                  maxDisputablePrice: maxDisputablePrice.toString(),
                  tokensLiquidated: liquidation.tokensLiquidated,
                  lockedCollateral: liquidation.lockedCollateral,
                  status: liquidation.status,
                  liquidationTimestamp: liquidationCreatedEvent.timestamp,
                  liquidationReceipt: liquidationCreatedEvent.tx_hash,
                };

```